lncfg/db.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/lightningnetwork/lnd/kvdb"
 	"github.com/lightningnetwork/lnd/kvdb/etcd"
+	"github.com/lightningnetwork/lnd/kvdb/postgres"
```

Comment:
I'd say the `lncfg` integration could be a separate commit.

Reply:
Done